### PR TITLE
[ResourceController] Use Symfony security first, than fallback to Sylius one

### PR DESCRIPTION
**Pull request by stloyd**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Fixed tickets | #2538
| License       | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php [hunk 1]
```diff
@@ -451,14 +451,22 @@ protected function handleView(View $view)
...
-            return true;
-        }
-
         $permission = $this->config->getPermission($permission);
-
-        if ($permission && !$this->get('sylius.authorization_checker')->isGranted(sprintf('%s.%s.%s', $this->config->getBundlePrefix(), $this->config->getResourceName(), $permission))) {
-            throw new AccessDeniedHttpException();
+        if ($permission) {
+            // Check access in Symfony security, if fail use Sylius one
+            if (!$this->get('security.context')->isGranted($permission)) {
```
Comment (amenophis) [inline src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php]: Would not it be better to remove the sylius.authorization_checker in favor of a Voter, so you can use the security.context for all cases ?

**Comment by geoffrey-brier** [comment]
:+1:

**Comment by stloyd** [comment]
Closing in favor of #2574.

File: src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php (hunk 1 above)
Comment (stloyd) [inline src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php]: Have a look at #2574.

**Pull request closed without merging**